extensions/answer_summarizers/models_test.py

```diff
@@ -40,6 +40,17 @@ class CalculationUnitTestBase(test_utils.GenericTestBase):
     def _create_answer_dict(
             self, answer, time_spent_in_card=3.2, session_id='sid1',
             classify_category=exp_domain.EXPLICIT_CLASSIFICATION):
+        """Returns the answer object.
+
+        Args:
+            answer: dict(str, *). The answer in dict format.
+            time_spent_in_card: float. The time spent (in sec) in each card.
+            session_id: str. The session id. By default, it's `sid1`.
```

Comment:
should we use single quotes here instead of `?

Reply:
Also, if we are mentioning default value here, do we need to do so for the other parameters too? (For consistency)

Reply:
Done